specutils/io/default_loaders/generic_cube.py

```diff
@@ -25,7 +27,7 @@ def identify_generic_fits(origin, *args, **kwargs):
             fits.getheader(args[0])['NAXIS'] == 3)
 
 
-@data_loader("cubetest1", identifier=identify_generic_fits)
+@data_loader("Cube", identifier=identify_generic_fits, extensions=['fits'])
```

Comment:
This should probably be adjusted to use `column_mapping`?  or if that's not obvious how to do, perhaps this can just do what I've suggested here to just "hide" it.  Otherwise we are accidentally turning what is obviously a "test" into something that appears finished (by naming it "Cube"), and this is definitely *not* sufficient for the huge world of spectral cubes.

```suggestion
# not yet ready because it's not generic enough and does not use 
#@data_loader("Cube", identifier=identify_generic_fits, extensions=['fits'])
```

Reply:
Noted, removing from circulation for now.